inception-project/inception
# #2567 - Problem with string feature editor initialization

**What's in the PR**
- Disable JQuery support for Boostrap to avoid the BS Tooltip clashing with the JQuery tooltip that we use e.g. in the Kendo dropdowns
- Fix sticky-dropdowns script to work without BS-JQuery integration
- Remove special handling of JQuery <-> BS during JS header adding in Wicket

**How to test manually**
* Try different parts of the UI (no specifics) and check if turning off JQuery support in BS might have unintended side-effects

**Automatic testing**

**Documentation**
